strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/pypi/PypiBrowsePackageHtmlResponseBuilderTest.java

```diff
@@ -0,0 +1,130 @@
+    }
+
+    @Test
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
+    public void testOnlyOnePackageFound(@PypiTestRepository(repositoryId = REPOSITORY_RELEASES, storageId = REPOSITORY_STORAGE) Repository repository,
+                                        @PypiTestArtifact(repositoryId = REPOSITORY_RELEASES, storageId = REPOSITORY_STORAGE, id = "hello_world", versions = "1.0.0") Path packagePath)
+        throws IOException
+    {
+
+        PypiArtifactCoordinates artifactCoordinates = (PypiArtifactCoordinates) RepositoryFiles.readCoordinates((RepositoryPath) packagePath.normalize());
+
+        String links = "<a href=\"" + "/storages/" + repository.getStorage().getId() + "/" + repository.getId()
+                + "/packages/" + artifactCoordinates.buildWheelPackageFileName() + "\">"
+                + artifactCoordinates.buildWheelPackageFileName() + "</a><br>\n";
+
+        String packageName = "hello-world";
+        String expectedHtmlResponse = "<html>\n"
+                + "        <head>\n"
+                + "            <title>Links for " + packageName + "</title>\n"
+                + "        </head>\n"
+                + "        <body>\n"
+                + "            <h1>Links for " + packageName + "</h1>\n"
+                + "                   " + links
+                + "        </body>\n"
+                + "    </html>";
+
+        List<Path> paths = new ArrayList<>();
+        paths.add(packagePath.normalize());
+        String htmlResponse = htmlResponseBuilder.getHtmlResponse(paths, packageName, repository);
+
+        assertThat(htmlResponse).isNotBlank().isEqualTo(expectedHtmlResponse);
+    }
+
+    @Test
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class, ArtifactManagementTestExecutionListener.class })
+    public void testMultiplePackagesFound(@PypiTestRepository(repositoryId = REPOSITORY_RELEASES, storageId = REPOSITORY_STORAGE) Repository repository,
+                                          @PypiTestArtifact(repositoryId = REPOSITORY_RELEASES, storageId = REPOSITORY_STORAGE, id = "hello_world", versions = { "1.0",
+                                                                                                                                                                 "2.0",
+                                                                                                                                                                 "3.0",
+                                                                                                                                                                 "4.0",
+                                                                                                                                                                 "5.0"
+                                          }) List<Path> packagePaths)
+        throws IOException
```

Comment:
Cosmetic: Would you mind changing this to:
```java
    public void testMultiplePackagesFound(@PypiTestRepository(repositoryId = REPOSITORY_RELEASES,
                                                              storageId = REPOSITORY_STORAGE)
                                          Repository repository,
                                          @PypiTestArtifact(storageId = REPOSITORY_STORAGE,
                                                            repositoryId = REPOSITORY_RELEASES,
                                                            id = "hello_world",
                                                            versions = { "1.0",
                                                                         "2.0",
                                                                         "3.0",
                                                                         "4.0",
                                                                         "5.0"})
                                          List<Path> packagePaths)
        throws IOException
```

(And apply the same sort of formatting across the rest of the pull request?)

Thanks! :)

Reply:
Noted and fixed.